clients/da-vinci-client/src/main/java/com/linkedin/davinci/ingestion/IsolatedIngestionBackend.java

```diff
@@ -1,5 +1,7 @@
 package com.linkedin.davinci.ingestion;
 
+import static com.linkedin.venice.ingestion.protocol.enums.IngestionCommandType.*;
```

Comment:
In general we want to avoid wildcard imports, although for the static import of enum values I guess the risk of misuse should be pretty low... Eventually we may implement a static analysis rule to prevent all wildcard imports though, in which case that would likely end up getting changed.

Reply:
I did not mean to do that but I found that IntelliJ does not save the trick config we use to avoid static import every time it shutdowns.. will change back.